components/bl-pdfViewer-component/styles/index.less

```diff
@@ -111,6 +111,10 @@
     background-color: #aaa;
   }
 
+  .no-data-text {
+    text-align: center;
```

Comment:
Also will be good to add some variables. 

color, font-size, text-align

And change this element from h3 to span. Or explain why that should be heading. Using it for styling purposes it's not a good idea.